frontend/src/index.html

```diff
@@ -7,6 +7,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
You could consider embedding the fonts in the app so that you don't have to fetch them from Google servers every time the application is loaded.

Reply:
Meh